packages/core/src/components/RadioButton/RadioButtonRow.tsx

```diff
@@ -46,24 +46,24 @@ const getRadioButtonAlignment = (
 };
 
 const renderLabel = (
-  value: string | React.ReactNode,
+  label: string | React.ReactNode,
   labelStyle: StyleProp<TextStyle>,
   textStyle: StyleProp<TextStyle>
 ) => {
-  if (typeof value === "string") {
-    return <Text style={[labelStyle, textStyle]}>{value}</Text>;
+  if (typeof label === "string") {
+    return <Text style={[labelStyle, textStyle]}>{label}</Text>;
   } else {
-    return <>{value}</>;
+    return <>{label}</>;
   }
 };
 
 const RadioButtonRow: React.FC<RadioButtonRowProps & IconSlot> = ({
   Icon,
   label,
-  value,
+  value = "",
   color,
   unselectedColor,
-  onPress = () => {},
+  onPress,
```

Comment:
why remove the default function?

Reply:
@peter Not needed.  if a real one is passed in it is called for real.  See line 87.